### DISCUSSION TITLE
Shortcuts to run the code

### DISCUSSION THREAD

**Discussion by aNewUsernameToMake** [GDQuest/learn-gdscript#162] [Ideas]
Hey there, just played the game, and I just want to ask if you can make some shortcuts for the code to run like in many IDEs? I don't like just lifting my hand off the keyboard and pressing run every time using my mouse, so I wanted shortcuts in the game.

**Comment by YuriSizov** [comment]
I'm not sure if it's in the live version yet, but you can in fact use your keyboard to operate the practice screen. Try <kbd>Ctrl + Enter</kbd> to run code (and hover over buttons to peek at their associated shortcuts).

**Reply by NathanLovato** [reply]
Yup, it's already in the staging version: Ctrl Enter or F5 to run the code.

**Reply by aNewUsernameToMake** [reply]
ok thanks
